test_launch_testing/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>test_launch_testing</name>
+  <version>0.7.3</version>
+  <description>Tests for the ROS launch testing suite.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <license>Apache License 2.0</license>
+  <author>Esteve Fernandez</author>
+
+  <build_depend>ament_cmake</build_depend>
+  <build_depend>ament_cmake_python</build_depend>
+
+  <exec_depend>ament_index_python</exec_depend>
+  <exec_depend>launch</exec_depend>
```

Comment:
Does this need exec depends anymore? Isn't it only tests?

Reply:
It is only tests. Dependencies removed.